src/applications/vaos/tests/components/ConfirmedAppointmentListItem.unit.spec.jsx

```diff
@@ -87,8 +87,8 @@ describe('VAOS <ConfirmedAppointmentListItem> Regular Appointment', () => {
     ).to.contain('C&P BEV AUDIO FTC1');
   });
 
-  it('should not show facility address', () => {
-    expect(tree.find('FacilityAddress').exists()).to.be.false;
+  it('should show facility address', () => {
```

Comment:
We won't have the facility address for confirmed VA appointments, just for confirmed CC appointments and requests. So I don't think this test should be different than before.

Reply:
Thanks for pointing this out. Reverted the test.